recipes/wildmidi/all/conanfile.py

```diff
@@ -0,0 +1,98 @@
+from conan import ConanFile
+from conan.tools.microsoft import is_msvc
+from conan.tools.files import export_conandata_patches, get, copy, rmdir
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+import os
+
+
+required_conan_version = ">=1.53.0"
+
+
+class WildmidiConan(ConanFile):
+    name = "wildmidi"
+    version = "0.4.5"
+    description = "The WildMIDI library"
+    license = ("GPL-3.0+", "LGPL-3.0")
```

Comment:
For the licenses, can you use the [SPDX identifiers](https://spdx.org/licenses/)?

Reply:
I thought I'm using them, there are "GPL-3.0+" and "LGPL-3.0" identifiers in the table. If I'm missing something, can you elaborate?

Reply:
Yes, I wasn't very clear. The `GPL-3.0+` and `LGPL-3.0` show up, but only in the deprecated table. You have to scroll up to the table heading to see that it's the "deprecated" identifiers list.

The updated identifiers would be `GPL-3.0-or-later` and `LGPL-3.0-only`.

However, since this recipe is only packaging the library and not the player, it only falls under the LGPL, so it should be fine to drop the GPL one unless you want to add an option to the recipe to build the player. In such a case, you can put _both_ licenses in the license attribute and set the license appropriately in the `configure` method.

You should probably handle the licenses that are copied in via the `package` method similarly.

Reply:
It's all clear now, thanks! I'm not packaging the player, so only LGPL license is packaged and set in conanfile